DEVELOPING.md

```diff
@@ -38,6 +38,16 @@ to a gcr.io repository before the integration tests can run.
 ```bash
 $ RUNTIME_IMAGE=gcr.io/my-project-id/openjdk:my-tag
 $ gcloud docker -- push $RUNTIME_IMAGE
-$ ./scripts/integration_test.sh $RUNTIME_IMAGE
```

Comment:
Let's leave this in for the case when people want to run both integration tests.
Also, in the the paragraph above, mention GKE in addition to App Engine.